test/exercises/Mandelbrot/solutions/mandelbrot4-taskpar.chpl

```diff
@@ -17,7 +17,7 @@ config const maxSteps = 50;
 // 
 // The number of tasks we should spawn.
 //
-config const numTasks = here.numCores; // Cores on locale #0.
+config const numTasks = here.numPUs(physical=true); // Cores on locale #0.
```

Comment:
I personally like just using `numPUs()`, but I think if you want to specify the value (and presumably want users to specify it as well) then it shouldn't have a default value.